src/api/app/views/layouts/webui/_unread_notifications_counter.html.haml

```diff
@@ -1,5 +1,5 @@
 .notifications-counter
   %i.fas.fa-bell
-  - if defined?(unread_notifications_count) && unread_notifications_count.positive?
+  - if defined?(unread_notifications_count) && unread_notifications_count&.positive?
```

Comment:
Both sides of the `&&` are the same

Reply:
mmm, what do you mean?
- the left side is a check in case nothing ever initialized the `defined?(unread_notifications_count)`, it prevents the error `undefined local variable or method unread_notifications_count`
- the right side is a check in case the `unread_notifications_count` is initialized but for any reason is set to `nil` before calling the `.positive?`, it prevents the error `undefined method `positive?' for nil:NilClass`

Unfortunately we experienced both cases, that's why we need to check for both